Gruntwork patcher

lubiarz-oliwia opened the pull request.

Commit:
basic structure for patcher page

Commit:
updated navbar, new hero

Commit:
fixed hero

Commit:
fixed after CR

klijakub committed:
Merge pull request #4 from Altalogy/feature/navbar-hero

updated navbar, new hero

klijakub committed:
Merge pull request #3 from Altalogy/feature/patcher-page-structure

Feature/patcher page structure

Commit:
created feature section

Commit:
Table section, Docs section

Commit:
slider logic, slider styles, RWD for slider section

Commit:
slider modified, how it works section - logic added, data set added, asciinema video added

Commit:
video section

Commit:
fixed buttons for how it works section

Commit:
added icon for video

klijakub commented:
@lubiarz-oliwia 
Export this image in higher quality 
<img width="1285" alt="image" src="https://user-images.githubusercontent.com/4997537/191508191-761fbafe-f8ba-4aa0-a8f2-fcd8d4ceb7ad.png">
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/4997537/191508515-4dc16496-7b30-4743-a492-eb1680fb583e.png">

klijakub commented on assets/css/global.scss:
```diff
@@ -1299,6 +1348,26 @@ $padding-section-vertical-desktop: 60px;
+    transform: translate(-50%);
+  }
+}
+
```
Should be in `patcher.scss`

klijakub commented on assets/css/pages/patcher.scss:
```diff
@@ -0,0 +1,419 @@
+      // Chrome
+      @media screen and (-webkit-min-device-pixel-ratio: 0) {
+        & {
+          // overflow: hidden;
```
Remove comment

klijakub commented on assets/css/pages/patcher.scss:
```diff
@@ -0,0 +1,419 @@
+        &::-webkit-slider-runnable-track {
+          height: 10px;
+          -webkit-appearance: none;
+          // margin-top: -1px;
```
Remove comment

klijakub commented on assets/css/pages/patcher.scss:
```diff
@@ -0,0 +1,419 @@
+          border-radius: 13px;
+        }
+
+        &:active::-webkit-slider-thumb {
+          //background: #1E2530;
+        }
```
Do we need this?

klijakub commented:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/4997537/191518695-631da09f-f1e4-4631-8f3f-2c03f6366807.png">
Icon isn't sharp, poor quality export

klijakub commented:
![image](https://user-images.githubusercontent.com/4997537/191524129-009caa8c-d0aa-4602-9c70-3aadabd6fc76.png)
Fix hover issue

klijakub commented:
Mobile view issue:
<img width="403" alt="image" src="https://user-images.githubusercontent.com/4997537/191527303-c48efe30-a139-48a4-b262-8296b80fc7ab.png">

klijakub commented on assets/js/asciinema-player.min.js:
```diff
@@ -0,0 +1 @@
+var AsciinemaPlayer=function(A){"use strict";function g(A,g,I,B,Q,C,E){try{var t=A[C](E),e=t.value}catch(A){return void I(A)}t.done?g(e):Promise.resolve(e).then(B,Q)}function I(A){return function(){var I=this,B=arguments;return new Promise((function(Q,C){var E=A.apply(I,B);function t(A){g(E,Q,C,t,e,"next",A)}function e(A){g(E,Q,C,t,e,"throw",A)}t(void 0)}))}}function B(A,g){if(!(A instanceof g))t ...
```
Remove console.debug

klijakub commented:
Export in better quality:
![image](https://user-images.githubusercontent.com/4997537/191534144-6ec6dc91-7b9e-443d-ab73-65b03eced5c3.png)

josh-padnick committed:
Fix job links.

josh-padnick committed:
Merge pull request #748 from gruntwork-io/fix-job-link

Fix job links.

lubiarz-oliwia committed:
fixed bugs

ArkuVonSymfon commented on _includes/scripts.html:
```diff
@@ -8,6 +8,55 @@
+  AsciinemaPlayer.create(
+    "{{ site.assets_base_url }}asciinema/Patcher.cast",
+    document.getElementById("patcher-player-cast-2"),
+    { cols: 101, rows: 24, autoPlay: true }
```
Was this one done on purpose with different cols and rows? Any others have the same like "151" and "36"

ArkuVonSymfon commented on _includes/styles.html:
```diff
@@ -1 +1,6 @@
+  rel="stylesheet"
+  type="text/css"
+  href="{{ site.assets_base_url }}css/asciinema-player.css"
+/>
```
end line

ArkuVonSymfon commented on _data/patcher.yml:
```diff
@@ -0,0 +1,18 @@
+  planned: Set up automatic updates with the web UI
+- name: Availability
+  beta: Part of the Gruntwork Subscription
+  planned: Standalone product
```
end line

ArkuVonSymfon commented on assets/js/slider.js:
```diff
@@ -0,0 +1,28 @@
+    $(".hours").html(calculatedHours);
+    $(".money-saved").html("$" + calculatedSavedMoney);
+  });
+});
```
end line

ArkuVonSymfon commented on pages/patcher/_hero.html:
```diff
@@ -0,0 +1,32 @@
+        style="z-index: 999; cursor: pointer; max-height: 505px;" />
+    </div>
+  </div>
+</div>
```
end line

ArkuVonSymfon commented on pages/patcher/_how-it-works.html:
```diff
@@ -0,0 +1,63 @@
+            document.querySelector('.without-patcher').classList.add('active-btn')
+        }
+    }
+</script>
```
end line

ArkuVonSymfon commented on pages/patcher/_patcher-table.html:
```diff
@@ -0,0 +1,47 @@
+      </div>
+    </div>
+  </div>
+</div>
```
end line

ArkuVonSymfon commented on pages/patcher/_video.html:
```diff
@@ -0,0 +1,26 @@
+            .querySelector(".video-wrapper")
+            .classList.toggle("video-wrapper-active");
+    }
+</script>
```
end line

ArkuVonSymfon commented on pages/patcher/_slider.html:
```diff
@@ -0,0 +1,73 @@
+        analysis.</a>
+    </p>
+  </div>
+</div>
```
end line

ebeneliason committed:
Fix social icon alignment

lubiarz-oliwia committed:
after code review, fixed tooltip for mobile

ebeneliason committed:
Merge pull request #749 from gruntwork-io/apt-2021-fix-social-icons

Fix social icon alignment

yorinasub17 committed:
Update team page to remove Yori (#750)

brikis98 committed:
Remove Rho :( (#751)

* Remove Rho :(

* Delete rho profile pic

lubiarz-oliwia committed:
Updated how it works section

lubiarz-oliwia committed:
added transition for tooltip

Commit:
Remove some Jenkins references

Commit:
Replace variants of 'about a day' with 'a few days'

Commit:
Reflect new limits on Ref Arch options

Commit:
Merge pull request #754 from gruntwork-io/ref-arch-emergency-measures

Reflect new Ref Arch emergency measures

Commit:
Remove Jenkins from Ref Arch pages (#753)

Commit:
Add "bio" and "where we work" for Andrew Ellison

zackproser committed:
Merge pull request #756 from gruntwork-io/andrew/add-to-team

Add "bio" and "where we work" for Andrew Ellison

brikis98 committed:
Add equity to careers page (#759)

Add it to the benefits list, as it is an important part of our compensation package.

brikis98 committed:
Add EU Engineering Manager role (#758)

* Add EU Engineering Manager role

Replace the EU Software Engineer role.

* Fix apply URL

arsci committed:
add ecs-task-scheduler to ecs iac library page

arsci committed:
Merge pull request #760 from gruntwork-io/feature/add-ecs-scheduling-module

Add new ecs-task-scheduler module to ECS IaC library page

eak12913 committed:
Revert "Add new ecs-task-scheduler module to ECS IaC library page" (#761)

eak12913 committed:
Revert "Revert "Add new ecs-task-scheduler module to ECS IaC library page" (#761)" (#763)

This reverts commit dad8ef3a96a3d1eb437ea03570e18562a20d233f.

arsci committed:
Add Ryan to team about page (#762)

* Add Ryan to team about page

techpink committed:
Add Jason to about page

techpink committed:
Merge pull request #764 from gruntwork-io/about-jason

Add Jason to about page

Commit:
Restore 'in about a day' messaging

Commit:
Merge pull request #765 from gruntwork-io/restore-about-a-day-msg

Restore 'in about a day' messaging

brikis98 committed:
Remove Ina :( (#767)

* Remove Ina :(

* Delete Ina's image

brikis98 committed:
Update Jim & Josh titles (#768)

eak12913 committed:
Updated the CircleCI job to use contexts instead of environment variables (#769)

This will make it easier to rotate credentials in the future.

eak12913 committed:
Create FUNDING.yml (#770)

iantybo committed:
Update website-terms.yml

iantybo committed:
Update terms-of-service.yml

iantybo committed:
Update website-terms.yml

iantybo committed:
Merge pull request #771 from gruntwork-io/ib/update-address

Update Gruntwork Address

MoonMoon1919 committed:
chore: add Max Moon to team

MoonMoon1919 committed:
Merge pull request #772 from gruntwork-io/chore/add-max-to-about-page

chore: add Max Moon to team

brikis98 committed:
EM job is filled (#773)

iantybo committed:
Update _form.html

iantybo committed:
Update _form.html

iantybo committed:
Update _form.html

iantybo committed:
Update _hero.html

iantybo committed:
Update _hero.html

iantybo committed:
Update _form.html

iantybo committed:
Update _hero.html

iantybo committed:
Update _hero.html

iantybo committed:
Merge pull request #774 from gruntwork-io/ib/update-contact-form

contact: update Contact Us form fields

lubiarz-oliwia committed:
Merge branch 'master' into gruntwork-patcher

lubiarz-oliwia committed:
Patcher updates
